deploy/svg_generator/svg_generator.py

```
@@ -84,84 +95,165 @@
+    if default_response is None:
+        default_response = {}
 
+    cache = await mc.get(key_b)
 
-async def get_local_time_formatted():
-    current_datetime = datetime.datetime.now(tz=server_timezone).strftime("%Y-%m-%d %H:%M:%S")
-    return current_datetime
+    if cache:
+        cache = json.loads(cache.decode("utf-8"))
+    else:
+        cache = default_response
```

Ruff (0.8.2)

Use ternary operator `cache = json.loads(cache.decode("utf-8")) if cache else default_response` instead of `if`-`else`-block

Replace `if`-`else`-block with `cache = json.loads(cache.decode("utf-8")) if cache else default_response`

(SIM108)